garrysmod/lua/vgui/dframe.lua

```diff
@@ -16,12 +16,15 @@ local PANEL = {}
 AccessorFunc( PANEL, "m_bIsMenuComponent", 		"IsMenu", 			FORCE_BOOL )
 AccessorFunc( PANEL, "m_bDraggable", 			"Draggable", 		FORCE_BOOL )
 AccessorFunc( PANEL, "m_bSizable", 				"Sizable", 			FORCE_BOOL )
+AccessorFunc( PANEL, "m_bMinMaxAnim", 			"MinMaxAnim", 		FORCE_BOOL )

```

Comment:
She's talking about the HTs that pad out the accessors.
@KateAdams I'm planning to clean the files up in the next commit.